draft-quic-atsss-reqs.mkd

```diff
@@ -118,7 +118,11 @@ The two scenarios that we consider in this document are:
 - the failure of one access network that forces all data to switch to the other access network
 - the possibility of splitting the transmission of the data over the two access networks even if they have different characteristics (delay, bandwidth, congestion)
 
+## Discovery of the MPTCP Proxy
```

Comment:
This paragraph interleaves three ideas.

1) There is a proxy mechanism discovery used for ATSSS
2) The proxy used is the "Transport Converter"
3) A set of steering policies define how flows are distributed

Only the first relates to the subsection title.